docs/tutorials/integrations/kserve/kserve-custommetrics-one.md

```diff
@@ -0,0 +1,117 @@
+---
+template: main.html
+---
+
+# Single Model Validation
+
+This tutorial shows how easy it is validate SLOs for a single model in [KServe](https://kserve.github.io/website/0.10/) when fetching metrics from a metrics database like Prometheus. We show this using the `sklearn-iris` model used for a [first `InferenceService`](https://kserve.github.io/website/master/get_started/first_isvc/) in the KServe documentation.
+
+???+ "Before you begin"
+    1. Try [your first experiment](../../../getting-started/your-first-experiment.md). Understand the main [concepts](../../../getting-started/concepts.md) behind Iter8 experiments.
+    2. Ensure that you have the [kubectl](https://kubernetes.io/docs/reference/kubectl/) CLI.
+    3. Have access to a cluster running [KServe](https://kserve.github.io/website). You can create a [KServe Quickstart](https://kserve.github.io/website/0.10/get_started/#before-you-begin) environment as follows:
+    ```shell
+    curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.10/hack/quick_install.sh" | bash
+    ```
+    These instructions can be used to [install Prometheus](https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring#install-prometheus).
```

Comment:
Call this out as step 4. And use the verb like 1, 2, and 3.

4. Install Prometheus monitoring for Serve [using these instructions](https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring#install-prometheus).

The above should help convey that Prometheus monitoring is a pre-req for this tutorial, just like 1, 2, and 3, and not an optional step.

Reply:
done